app/controllers/api/v1/positions_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Api::V1
+  # Controller for Positions
+  class PositionsController < ApplicationController
+    before_action :set_position, only: %i[show update destroy]
+
+    # GET /positions
+    def index
+      @positions = Position.order(:id)
+
+      render json: @positions
+    end
+
+    # GET /positions/1
+    def show
+      render json: @position
+    end
+
+    # POST /positions
+    def create
+      @position = Position.new(position_params)
+
+      if @position.save
+        render json: @position, status: :created
+      else
+        render json: @position.errors, status: :unprocessable_entity
+      end
+    end
+
+    # PATCH/PUT /positions/1
+    def update
+      if @position.update(position_params)
+        render json: @position
+      else
+        render json: @position.errors, status: :unprocessable_entity
+      end
+    end
+
+    # DELETE /positions/1
+    def destroy
+      @position.destroy
```

Comment:
Is destroy safe here? Should we be hard deleting?

Reply:
@reidka what do you think?

Reply:
I used to be very hesitant to delete anything from the database, but I think in this context it will be useful.  I'm thinking of the case where we upload some positions and then realize that we uploaded the wrong ones.

We will want to make sure that there are no other connections to the position before deleting.  In particular, no "assignments".  Primarily for database consistency.